### ISSUE TITLE
Crash occurs when attempting to morph path strings that begin with leading spaces

### ISSUE THREAD

**Issue by alexjlockwood** [pbeshai/d3-interpolate-path#17]
For example, morphing the two strings results in the below crash:

```
// Note the leading space in path1.
const path1 = ` M150,0 L200,40 L100,60 L120,80 L50,100 L250,150 L120,200`;
const path2 = `M120,0 L100,30 L20,45 L220,60 L270,90 L180,95 L50,130 L85,140 L140,150 L190,175 L60,195`;
```

Stack trace:

```
d3-interpolate-path.js:213 Uncaught TypeError: Cannot read property 'toUpperCase' of undefined
    at commandToObject (d3-interpolate-path.js:213)
    at Array.map (<anonymous>)
    at Object.interpolatePath (d3-interpolate-path.js:495)
    at SVGPathElement.<anonymous> (index.js:125)
    at SVGPathElement.e (d3v4+jetpack.js:1)
    at r (d3v4+jetpack.js:1)
    at d3v4+jetpack.js:1
    at pn (d3v4+jetpack.js:1)
    at dn (d3v4+jetpack.js:1)
```

**Comment by alexjlockwood** [comment]
The crash also occurs with leading enters/other whitespace too.

**Comment by pbeshai** [comment]
Thanks, I'll look into it!

**Comment by pbeshai** [comment]
Fixed in v2.0.1
